ddnet-release.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
why make it bash?

Reply:
Seems like "&>" is not a POSIX shell feature and it is used in this script. So on some setups this can fail.